src/DataTableColumn.php

```diff
@@ -79,7 +79,7 @@ public function init()
             throw new InvalidConfigException("Either 'data' or 'render' properties must be specified.");
         }
 
-        if ($this->title === null) {
+        if ($this->title === null && !is_null($this->attribute)) {
```

Comment:
Did you catch some errors here?

Reply:
Yes, there was an error with the LinkColumn class where this property was null and generated an error.

Basicly in some places of the code I was a bit confues why take the property `attribute` (in setAttribute()) but store it internal as `$data`.

So I yust added a chack to avoid the error and left the rest like it is.